DevoxxClientMobile/src/main/java/com/devoxx/views/SessionPresenter.java

```diff
@@ -286,7 +286,12 @@ private void loadAuthenticatedNotesView(final Session session) {
             company.setWrapText(true);
             GridPane.setHgrow(company, Priority.ALWAYS);
 
-            Label summary = new Label(speaker.getSummary());
+            Label summary = new DataLabel();
```

Comment:
We only need a DataLabel when `speaker.isDetailsRetrieved()` returns `false`?

Reply:
DataLabel is a Label with `graphicProperty` set as a ProgressIndicator until its `textProperty` is updated. So, when `isDetailsRetrieved` is true, it will act as just a Label.

Reply:
I only mentioned it because it will still set the progress indicator first and add the change listener on textProperty. This is not really necessary if you set the text directly afterwards?

Reply:
Good point. Indeed, I can use a Label when `isDetailsRetrieved` is true.

I have also been thinking of how to make `DataLabel` as close to `Label` as possible, so that we don't need to worry about conditions like this.

One of the things would be to remove the listener as soon as the text changes.